pyproject.toml

```diff
@@ -39,44 +39,43 @@ optional = true
 
 [tool.poetry.group.format.dependencies]
 black = "^24.3.0"
-isort = "^5.12.0"
+isort = "^5.13.2"
 
 [tool.poetry.group.lint]
 optional = true
 
 [tool.poetry.group.lint.dependencies]
+boto3 = "^1.34.82"
 black = "^24.3.0"
-isort = "^5.12.0"
-flake8 = "^6.0.0"
+isort = "^5.13.2"
+flake8 = "^7.0.0"
 flake8-docstrings = "^1.7.0"
 flake8-copyright = "^0.2.4"
-flake8-builtins = "^2.1.0"
-pyproject-flake8 = "^6.0.0.post1"
+flake8-builtins = "^2.5.0"
+pyproject-flake8 = "^7.0.0"
 pep8-naming = "^0.13.3"
-codespell = "^2.2.5"
-shellcheck-py = "^0.9.0.5"
+codespell = "^2.2.6"
+shellcheck-py = "^0.10.0.1"
 
 [tool.poetry.group.unit.dependencies]
-pytest = "^7.4.0"
-pytest-asyncio = "<0.23"
-coverage = {extras = ["toml"], version = "^7.4.1"}
+pytest = "^8.1.1"
+pytest-asyncio = "*"
```

Comment:
This should only be needed to suppress warnings. We should just use whatever version `pytest-operator` is pulling

Reply:
Agreed